bundle/manifests/amalthea.clusterserviceversion.yaml

```diff
@@ -21,8 +21,8 @@ metadata:
         }
       ]
     capabilities: Basic Install
-    createdAt: "2024-11-01T02:52:56Z"
-    operators.operatorframework.io/builder: operator-sdk-v1.36.1
+    createdAt: "2024-12-09T08:25:56Z"
```

Comment:
The changes here are just a reordering and consolidation. Nothing truly changes.